### PR TITLE
Added commands and services for the developer.overheid apis and repositories sync

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

File: Service/DeveloperOverheidService.php [hunk 1]
```diff
@@ -64,132 +65,215 @@ public function setStyle(SymfonyStyle $io): self
...
 
-        $this->io->debug("Found ".count($repositories)." repositories");
-        foreach($repositories as $repository){
-            $result[] = $this->importRepository($repository);
+        // rows per page are 10, so i get only 10 results
+        $response = $this->callService->call($source, '/apis');
+
+        $components = json_decode($response->getBody()->getContents(), true);
+
+        $this->io->success("Found ".count($components)." components");
```
Comment (rjzondervan) [inline Service/DeveloperOverheidService.php]: Didn't StyleCI change this for '?

**Review by rjzondervan — APPROVED** [review]
Looks good. StyleCI might change some quote-marks

**Review by bbrands02 — APPROVED** [review]

File: Command/DeveloperOverheidGetComponentsCommand.php [hunk 2]
```diff
@@ -2,40 +2,50 @@
...
     }
 
     protected function configure(): void
     {
         $this
-            ->setDescription('This command triggers OpenCatalogi FindGithubRepositoryThroughOrganizationService')
-            ->setHelp('This command allows you to update a organizations with found opencatalogi.yml info');
+            ->setDescription('This command triggers OpenCatalogi DeveloperOverheidService')
+            ->setHelp('This command allows you to get all components or one component from developer.overheid.nl/apis')
+            ->addOption('component', 'c', InputOption::VALUE_OPTIONAL, 'Get a singe component by id or name');
```
Comment (rubenvdlinde) [inline Command/DeveloperOverheidGetComponentsCommand.php]: *Single

**Review by rubenvdlinde — APPROVED** [review]

**Review by WilcoLouwerse — COMMENTED** [review]
We write to $this->io but never check if it isset or not. 
If we ever reach some of this code without first going through an execute() function where we set this-io it will not be set and break.

**Review by WilcoLouwerse — APPROVED** [review]

**Review by rubenvdlinde — APPROVED** [review]

**Pull request merged**